jobs/src/main/scala/dcos/metronome/MetronomeBuildInfo.scala

```diff
@@ -14,7 +14,7 @@ import scala.util.{ Failure, Success, Try }
   * Provides build information details regarding Metronome and Marathon at runtime.
   */
 case object MetronomeBuildInfo {
-  private val metronomeJar = "\\bmesosphere\\.metronome\\.metronome-[0-9.]+".r
```

Comment:
does this work locally? can we write a test for this?

Reply:
it does work locally... I did test it.   I don't see how to write a test for it (it's a chicken and egg problem...  the artifact isn't built before tests and the tests would need the artifact to test).   I suppose we could mock something but I'm not sure it would really add that much value.   There are lots of ways it could break that a sim wouldn't catch (like not creating the manifest file).   What exactly are you wanting to test?   the regex match?

Reply:
no the fact that we output meaningful version so that we don't release it in a broken form as we did before :)